resources/assets/js/app/models/VoteAIModel.js

```diff
@@ -11,9 +13,10 @@ module.exports = BaseModel.extend({
 
         if (!_.isEmpty(errors)) {
             return errors;
-        }
+        } else this.trigger('valid');
```

Comment:
Feel free to use braces even if you have just one line in body of condition. It allows read code easily